app/Console/Commands/CreateUser.php

```diff
@@ -77,6 +78,8 @@ public function handle(Dispatcher $dispatcher, Validation $validation)
 
         $send_email = (!$this->option('no-email'));
 
+        $arguments['is_admin']   = (!$this->option('admin') ? 0 : 1);
```

Comment:
Could you change this to `$arguments['is_admin']   = (!$this->option('admin'));` as with the previous line

Reply:
Done